REFERENCE.md

```diff
@@ -370,6 +354,19 @@ class { 'stdlib::manage':
     }
 ```
 
+##### 
```

Comment:
It would be nice to have a title for this, but it looks like other examples lack one too.